src/main/java/org/spongepowered/api/event/entity/living/player/PlayerDeathEvent.java

```diff
@@ -53,7 +53,7 @@
      *
      * @return Whether the player keeps their inventory on death
      */
-    boolean keepsInventory();
+    boolean getKeepsInventory();
```

Comment:
Is this necessary? Is there no way to generate the event class without the prefix?